lib/resty/worker/events.lua

```diff
@@ -272,15 +272,14 @@ _M.post = function(source, event, data, unique)
     err = 'failed posting event "'..event..'" by "'..
           source..'"; '..tostring(err)
     log(ERR, "worker-events: ", err)
-    return success, err
   end
 
-  return _M.poll()
+  return success, err
```

Comment:
if it is to return `true` then this should probably be `return true, err` (to prevent it from returning the internal event_id)

Reply:
Done!